<?xml version="1.0"?>
<launch>
    <arg name="logoutput" default="screen"/>
    <rosparam command="delete" param="/RoverSmach" />
    <rosparam command="load" file="$(find rover20_state_mach)/config/smach_config.yaml" />
    <node name="state_machine" pkg="rover20_state_mach" type="state_machine20.py" output="$(arg logoutput)"/>
</launch>
